sim/druid/claw.go

```diff
@@ -0,0 +1,80 @@
+package druid
+
+import (
+	"time"
+
+	"github.com/wowsims/classic/sim/core"
+)
+
+func (druid *Druid) registerClawSpell() {
+	flatDamageBonus := map[int32]float64{
+		20: 27,
+		28: 39,
+		38: 57,
+		48: 88,
+		60: 115,
+	}[druid.Level]
+
+	druid.Claw = druid.RegisterSpell(Cat, core.SpellConfig{
+		SpellCode: SpellCode_DruidClaw,
+		ActionID: core.ActionID{SpellID: map[int32]int32{
+			20: 1082,
+			28: 3029,
+			38: 5201,
+			48: 9849,
+			60: 9850,
+		}[druid.Level]},
+		SpellSchool: core.SpellSchoolPhysical,
+		DefenseType: core.DefenseTypeMelee,
+		ProcMask:    core.ProcMaskMeleeMHSpecial,
+		Flags:       core.SpellFlagMeleeMetrics | core.SpellFlagAPL | SpellFlagOmen | SpellFlagBuilder,
+
+		EnergyCost: core.EnergyCostOptions{
+			Cost:   45 - 1*float64(druid.Talents.Ferocity),
```

Comment:
Should also update Idol of Ferocity in druid/items.go to update the cost of Claw as well, since it's the only Idol that actually does anything in Cat Form.

See the [example](https://github.com/wowsims/sod/blob/d5c9752ce47362686eb9849a4619a94448a79299/sim/druid/items.go#L186) from the SoD repo.

Reply:
Updated Idol of Ferocity to lower cost of Claw